src/cli/rules.py

```
@@ -217,6 +220,11 @@ def rules_export(
     content = yaml.dump(rs.to_dict(), default_flow_style=False, sort_keys=False)
 
     if output:
+        # A.cli: output file may not exist (we're creating it). The parent
+        # directory must exist though — resolve + sanity-check the parent.
+        output = resolve_cli_path(output, must_exist=False, must_be_dir=False)
+        if not output.parent.exists():
+            raise typer.BadParameter(f"Output directory does not exist: {output.parent}")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)